src/test/java/com/fredsonchaves07/moviecatchapi/api/services/mail/SendEmailTest.java

```diff
@@ -17,45 +17,34 @@ public class SendEmailTest {
     public void shouldSendEmail() {
         String email = "user@email.com";
         String subject = "Teste de envio de email";
-        String content = "Testando envio de email";
-        assertDoesNotThrow(() -> sendEmailService.send(new MessageEmailDTO(subject, email, content)));
+        assertDoesNotThrow(() -> sendEmailService.send(
+                new MessageEmailDTO(subject, email), null, null)
+        );
     }
 
     @Test
     public void notShouldSendEmailIfEmailIsNull() {
         String subject = "Teste de envio de email";
-        String content = "Testando envio de email";
         assertThrows(
                 SendEmailException.class,
-                () -> sendEmailService.send(new MessageEmailDTO(subject, null, content))
+                () -> sendEmailService.send(new MessageEmailDTO(subject, null), null, null)
         );
     }
 
     @Test
     public void notShouldSendEmailIfSubjectIsNull() {
         String email = "user@email.com";
-        String content = "Testando envio de email";
-        assertThrows(
-                SendEmailException.class,
-                () -> sendEmailService.send(new MessageEmailDTO(null, email, content))
-        );
-    }
-
-    @Test
-    public void notShouldSendEmailIfContentIsNull() {
-        String email = "user@email.com";
-        String subject = "Teste de envio de email";
         assertThrows(
                 SendEmailException.class,
-                () -> sendEmailService.send(new MessageEmailDTO(subject, email, null))
+                () -> sendEmailService.send(new MessageEmailDTO(null, email), null, null)
```

Comment:
Refactor to something like TemplateMailDTO to center the email template information in a specific class